numba_dppy/spirv_generator.py

```diff
@@ -170,24 +164,46 @@ def finalize(self):
         spirv_path = self._track_temp_file("generated-spirv")
 
         binary_paths = [spirv_path]
-        llvm_spirv_args = None
+
+        llvm_spirv_args = []
         for key in list(self.context.extra_compile_options.keys()):
             if key == LINK_ATOMIC:
                 from .ocl.atomics import get_atomic_spirv_path
 
                 binary_paths.append(get_atomic_spirv_path())
-            if key == LLVM_SPIRV_ARGS:
+            elif key == LLVM_SPIRV_ARGS:
```

Comment:
Are these two options mutually exclusive?

Reply:
No, but key will never be both.